### ISSUE TITLE
setAlias() not working

### ISSUE THREAD

**Issue by razorsharpshady** [yidas/codeigniter-model#23]
`setAlias()` doesn't seems to set alias for the table. Database throws error *unknown table p*.
Example:
```
 $this->paf->setAlias('p')->find()
        ->join('co_user AS co', 'co.user_id = p.user_id')
            ->select('p.*')->get()->result();
```

**Comment by yidas** [comment]
Hi @razorsharpshady,

This issue has been fixed from [version 2.18.0](https://github.com/yidas/codeigniter-model/releases/tag/2.18.0).

Thanks for the report!

**Comment by razorsharpshady** [comment]
Hey thanks for the fix Nick, Great job.
